test/functional/tools/explicit_conversion.xml

```diff
@@ -0,0 +1,23 @@
+<tool id="explicit_conversion" name="explicit_conversion">
+  <command>
+    cut -f 1 '${input1_table}' > 'col_output'
```

Comment:
I would argue that the from_work_dir needs an additional copy/move operation.
Not really needed for this PR, but I start to use this test more and more in workshops to show people examples ;)